src/item.rs

```diff
@@ -5,6 +5,19 @@ use crate::common::*;
 pub(crate) enum Item<'src> {
   Alias(Alias<'src, Name<'src>>),
   Assignment(Assignment<'src>),
+  Comment(Vec<&'src str>),
```

Comment:
Is there a reason that comments are stored together, i.e. as `Comment(Vec<&'src str>)` instead of `Comment(&'src str)`?

Reply:
Withouth `Vec` it is definitely cleaner.